spec/features/sign_in_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+describe "Sign in flow" do
```

Comment:
Rather than `describe` and `it` in feature specs, favor using `feature` and `scenario`.